src/main/java/lotto/domain/LottoMachine.java

```diff
@@ -0,0 +1,45 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoMachine {
+
+    private static final int MONEY_UNIT = 1000;
+
+    private final NumberGenerator numberGenerator;
+
+    public LottoMachine(NumberGenerator numberGenerator) {
+        this.numberGenerator = numberGenerator;
+    }
+
+    public Lottos createLottos(Money money) {
+        int count = getTryCount(money);
+        List<Lotto> lottos = new ArrayList<>();
+
+        for (int tried = 0; tried < count; tried++) {
+            Lotto lotto = createLotto();
+            lottos.add(lotto);
+        }
+
+        return new Lottos(lottos);
+    }
+
+    private Lotto createLotto() {
+        List<Integer> numbers = numberGenerator.generateNonDuplicated(LottoConstants.MIN_NUMBER,
+                LottoConstants.MAX_NUMBER, LottoConstants.NUMBER_COUNT);
+        return new Lotto(numbers);
+    }
+
+    private int getTryCount(Money money) {
+        if (!money.canDivideWith(MONEY_UNIT)) {
```

Comment:
```suggestion
        if (money.canNotDivideWith(MONEY_UNIT)) {
```
부정 연산자를 사용하는것보다 메서드명을 통해서 표현하는게 더 좋을것 같아요
```
money가 1000으로 나눠진다 -> false(나눠지지 않는다) -> ! -> true
vs
money가 1000으로 나눠지지 않는다
```